packages/teams-js/src/internal/communication.ts

```diff
@@ -611,42 +621,49 @@ const handleParentMessageLogger = communicationLogger.extend('handleParentMessag
 function handleParentMessage(evt: DOMMessageEvent): void {
   const logger = handleParentMessageLogger;
 
-  if ('id' in evt.data && typeof evt.data.id === 'number') {
+  if ('id' in evt.data && (typeof evt.data.id === 'number' || typeof evt.data.id === 'string')) {
```

Comment:
Hmm good point, my thought process is since this change affects all of our codebase we should have to run the unit tests twice, once in a legacy mode that sends numbers as it used to and the other that sends uuids.

I should add some unit tests in communication though that verifies that the `handleParentMessage` works correctly though.